chrome_manifest_v3/model.js

```
@@ -1,784 +1,764 @@
-                const cardId_k = this.pos === "at" ? this.cardId : "%cardId%"; // Won't know until we store the initial card
-                args.property = "cards/" + cardId_k + "/" + args.property;
-                this.data.push(args);
-            }
-        }
-        return this;
-    },
+    if (!this.cardId || this.cardId.length < 1 || this.cardId === '-1') {
+      pos = 'top';
+    } else {
+      const position_k = args.position || 'below';
+      const cardPos_k = parseInt(args.cardPos || 0, 10);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)